omnibus/files/private-chef-cookbooks/private-chef/templates/default/pg_hba.conf.erb

```diff
@@ -64,6 +64,8 @@
 # "local" is for Unix domain socket connections only
 local   all         <%= node['private_chef']['postgresql']['username'] %>                               peer
 
+host    all         <%= node['private_chef']['postgresql']['db_superuser'] %>         127.0.0.1/32           md5
```

Comment:
It looks like we could use samehost, or in theory localhost should work.

pg docs --
You can also write all to match any IP address, samehost to match any of the server's own IP addresses, or samenet to match any address in any subnet that the server is directly connected to.